<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SharpeningFilter</class>
 <widget class="QDialog" name="SharpeningFilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>226</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SharpeningFilter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Presets:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mPresetsCB"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="mKernelValuesLayout">
        <item row="2" column="3">
         <widget class="QLineEdit" name="mLineEditSE">
          <property name="text">
           <string>-1</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="mLineEditNW">
          <property name="text">
           <string>-1</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="mLineEditS">
          <property name="text">
           <string>-1</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="mLineEditCenter">
          <property name="text">
           <string>16</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLineEdit" name="mLineEditE">
          <property name="text">
           <string>-1</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLineEdit" name="mLineEditNE">
          <property name="text">
           <string>-1</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="mLineEditN">
          <property name="text">
           <string>-1</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="mLineEditW">
          <property name="text">
           <string>-1</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="mLineEditSW">
          <property name="text">
           <string>-1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Divider:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mDividerTB">
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="mOnlyOnMipmapsCheckbox">
        <property name="toolTip">
         <string>The user might not want to apply this on the main texture
Check this if you only want the sharpening filter to apply on the mipmaps</string>
        </property>
        <property name="text">
         <string>Only apply on mipmaps</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mOkBtn">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>mPresetsCB</tabstop>
  <tabstop>mLineEditNW</tabstop>
  <tabstop>mLineEditN</tabstop>
  <tabstop>mLineEditNE</tabstop>
  <tabstop>mLineEditW</tabstop>
  <tabstop>mLineEditCenter</tabstop>
  <tabstop>mLineEditE</tabstop>
  <tabstop>mLineEditSW</tabstop>
  <tabstop>mLineEditS</tabstop>
  <tabstop>mLineEditSE</tabstop>
  <tabstop>mDividerTB</tabstop>
  <tabstop>mOnlyOnMipmapsCheckbox</tabstop>
  <tabstop>mOkBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
